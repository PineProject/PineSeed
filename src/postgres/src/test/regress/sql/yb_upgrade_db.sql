--
-- Test case for unsupported collations
--
CREATE TABLE collate_test_fail(b text COLLATE "nds-x-icu");

CREATE COLLATION nds from "nds-x-icu";

CREATE TABLE collate_test_fail (a int, b text);
SELECT * from collate_test_fail where b like 'a'  COLLATE "nds-x-icu";
