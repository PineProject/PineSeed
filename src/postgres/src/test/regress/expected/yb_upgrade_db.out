--
-- Test case for unsupported collations
--
CREATE TABLE collate_test_fail(b text COLLATE "nds-x-icu");
WARNING:  collation is deprecated
LINE 1: CREATE TABLE collate_test_fail(b text COLLATE "nds-x-icu");
                                              ^
ERROR:  collation no longer supported and will be dropped in next release
LINE 1: CREATE TABLE collate_test_fail(b text COLLATE "nds-x-icu");
                                              ^
CREATE COLLATION nds from "nds-x-icu";
ERROR:  cannot create collation based on deprecated collation
CREATE TABLE collate_test_fail (a int, b text);
SELECT * from collate_test_fail where b like 'a'  COLLATE "nds-x-icu";
WARNING:  collation is deprecated
LINE 1: SELECT * from collate_test_fail where b like 'a'  COLLATE "n...
                                                          ^
 a | b
---+---
(0 rows)
